examples/purchaseTesterTypescript/android/gradle.properties

```diff
@@ -10,7 +10,7 @@
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
 # Default value: -Xmx512m -XX:MaxMetaspaceSize=256m
-org.gradle.jvmargs=-Xmx4096m -XX:MaxPermSize=4096m -XX:+HeapDumpOnOutOfMemoryError
```

Comment:
Was this causing any issues?

Reply:
@tonidero `MaxPermSize` is obsolete in Java 17 🤷‍♂️